packages/theme-check-common/src/checks/asset-url-filters/index.ts

```diff
@@ -1,11 +1,26 @@
-import { HtmlRawNode, HtmlVoidElement, LiquidFilter, NodeTypes } from '@shopify/liquid-html-parser';
-import { LiquidCheckDefinition, LiquidHtmlNode, Severity, SourceCodeType } from '../../types';
+import {
```

Comment:
We might want to rename the file from `asset-url-filters` to `remote-asset`, and also rename the check from AssetUrl to RemoteAsset.